packages/repository/src/__tests__/acceptance/has-one.relation.acceptance.ts

```diff
@@ -115,6 +115,70 @@ describe('hasOne relation', () => {
     ).to.be.rejectedWith(EntityNotFoundError);
   });
 
+  it('can PATCH hasOne instances', async () => {
+    existingCustomerId = 11;
+    const address = await controller.createCustomerAddress(existingCustomerId, {
+      street: '1 Amedee Bonnet',
+      zipcode: '69740',
+      city: 'Genas',
+      province: 'Rhone',
+    });
+
+    const persisted = await addressRepo.findById(address.zipcode);
+    expect(persisted.toObject()).to.deepEqual(address.toObject());
```

Comment:
Can we remove these two lines please? We have other tests in place to verify that `createCustomerAddress` works as expected. This test should focus on verifying PATCH behavior only.

Reply:
Ok, no problem.